src/Microsoft.AspNetCore.Mvc.TagHelpers/Microsoft.AspNetCore.Mvc.TagHelpers.csproj

```diff
@@ -16,6 +16,7 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.AspNetCore.Mvc.Razor\Microsoft.AspNetCore.Mvc.Razor.csproj" />
 
+    <PackageReference Include="Microsoft.AspNetCore.Razor" Version="$(AspNetCoreVersion)" />
```

Comment:
These assemblies are still needed for Tag Helpers. What's important is that we need to remove usage of the old compiler types.